backend/app/views/spree/admin/products/_form.html.erb

```diff
@@ -32,7 +32,7 @@
       <div data-hook="admin_product_form_price">
         <%= f.field_container :price do %>
           <%= f.label :price, class: 'required' %>
-          <%= render "spree/admin/shared/number_with_currency", f: f, amount_attr: :price,
+          <%= render "spree/admin/shared/number_with_currency", f: f, amount_attr: :price, required: true,
```

Comment:
We have a `Spree::Config.require_master_price` (which I don't know how widely is used but exists).

Can we change this line to `required: Spree::Config.require_master_price`?